bin/server.js

```diff
@@ -9,7 +9,7 @@ wss.on('connection', function connection(ws) {
     ws.on('message', function incoming(message) {
 
         let msg = JSON.parse(message);
-        console.log('received: %s', msg);
+        console.log('received: %s', JSON.stringify(msg));
```

Comment:
This means showing the actual JSON object in console instead of `[object Object]`